### DISCUSSION TITLE
Button with as="a" attribute not showing the externalLink Icon

### DISCUSSION THREAD

**Discussion by chjpark** [twilio-labs/paste#1758] [Q&A] [answered]
Hi,

I'm trying to use the button as a link to a new page and I'd like to have it render with the externalIcon over the left arrow icon. How can I do that? I checked the docs here: https://paste.twilio.design/components/button/ and for both, it shows to add as="a" in the attributes. What do I need to pass into the Button component in order to render it with the externalLink Icon?

Here's the screenshot of the current docs:
![Uploading Screen Shot 2021-08-10 at 12.44.21 PM.png…]()

**Comment by richbachman** [comment, marked as answer]
Hi @chjpark what version of core or button do you have in your `package.json`?

**Reply by chjpark** [reply]
I'm using 9.0.1

**Reply by richbachman** [reply]
Thanks! That change was introduced in core version 9.0.2, so if you update to that version you should be good to go.
